src/main/java/io/mo/db/Executor.java

```diff
@@ -110,7 +110,8 @@ public static void run(TestScript script){
                 if (command.checkResult()) {
                     script.addSuccessCmd(command);
                     command.getTestResult().setResult(RESULT.RESULT_TYPE_PASS);
-                    LOG.debug("[" + script.getFileName() + "][row:" + command.getPosition() + "][" + command.getCommand().trim() + "] was executed successfully");
+                    LOG.debug("[" + script.getFileName() + "][row:" + command.getPosition() + "][" + command.getCommand().trim() + "] was executed successfully, con[id="
```

Comment:
Why don't you use `String.format()`

Reply:
just because I did not use String.format() at the beginning , OK, it is a good idea, but next time, I modify this